### ISSUE TITLE
Is it open source for license use

### ISSUE THREAD

**Issue by shahzadraja** [MMAUG/myayphyu#4]
Can we use it for making our professional use?

**Comment by trhura** [comment]
Yes. Please assume it under a permissive license (MIT / BSD) and feel free to use for commercial / professional purposes.
